examples/Atlas/+atlasUtil/plotWalkingTraj.m

```diff
@@ -58,7 +58,7 @@
   qd=xt(nq+(1:nq));
   qdd=qddtraj.eval(t);
 
-  kinsol = doKinematics(r,q);
+  kinsol = doKinematics(r, q, qd);
```

Comment:
I made an error in this line in pull request #961 , Twan fixed this for me since we are computing Jdotv later